package.json

```diff
@@ -17,12 +17,12 @@
     "build:icons": "node --experimental-modules internals/populate-icons.js"
   },
   "peerDependencies": {
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1"
+    "react": "^17.0.1",
```

Comment:
I think here we need to add in 16 and 17 as a lot of our apps in supabase are still on react 16.

something like..
```
"peerDependencies": {
    "react": "^16.13.1 || ^17.0.1",
    "react-dom": "^16.13.1 || ^17.0.1"
  },
  ```
 
 i'll need to test this locally as well, make sure it doesn't break anything.

Reply:
Sounds good, will update it.